troposphere/static/js/modals/badge/ShowHelp.js

```diff
@@ -0,0 +1,15 @@
+define(function (require) {
+  "use strict";
+
+  var actions = require('actions'),
+      ModalHelpers = require('components/modals/ModalHelpers'),
+      BadgeHelp = require('components/modals/BadgeHelp.react');
```

Comment:
Can `actions` be removed here? 

Also - can we convert this to [ES6 modules](https://gist.github.com/lenards/711efb16001d02152598)?